<launch>
  <arg name="viewer_type" default="image_viewer"/>
  <arg name="camera_id" default="/"/>
  <arg name="image_src" default="/image_raw"/>

  <node pkg="viewers" name="$(arg viewer_type)" type="$(arg viewer_type)" output="screen">
    <param name="image_raw_topic" type="str" value="$(arg camera_id)$(arg image_src)"/>
  </node>

</launch>
